From 0373c43b2b9e907d919fd2bc8238b7f296a71341 Mon Sep 17 00:00:00 2001
From: Jason Liu <r64343@freescale.com>
Date: Thu, 5 Apr 2012 15:29:57 +0800
Subject: [PATCH 1271/2508] ENGR00178951-2 SYSRQ: enable CONFIG_MAGIC_SYSRQ by
 default

SYSRQ is very useful for kernel debug thus enable it by default.

SYSRQ support serial port, we can send the command via minicom:

CTRL A + F (send BRK) + T: to dump the task information

Enable SYSRQ by default will not involve any performance drop

Signed-off-by: Jason Liu <r64343@freescale.com>
---
 arch/arm/configs/imx6_defconfig |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/arm/configs/imx6_defconfig b/arch/arm/configs/imx6_defconfig
index eb2e332..85bdfca 100644
--- a/arch/arm/configs/imx6_defconfig
+++ b/arch/arm/configs/imx6_defconfig
@@ -2482,7 +2482,7 @@ CONFIG_DEFAULT_MESSAGE_LOGLEVEL=4
 CONFIG_ENABLE_WARN_DEPRECATED=y
 CONFIG_ENABLE_MUST_CHECK=y
 CONFIG_FRAME_WARN=1024
-# CONFIG_MAGIC_SYSRQ is not set
+CONFIG_MAGIC_SYSRQ=y
 # CONFIG_STRIP_ASM_SYMS is not set
 # CONFIG_UNUSED_SYMBOLS is not set
 CONFIG_DEBUG_FS=y
-- 
1.7.7.4

